pkg/interface/src/views/apps/chat/components/lib/ChatMessage.tsx

```diff
@@ -116,15 +112,20 @@ export default class ChatMessage extends Component<ChatMessageProps> {
       scrollWindow
     };
 
+    const unreadContainerStyle = {
+      transition: '.25s height ease-in-out',
+      height: isLastRead ? '1.66em' : '0',
+    };
+
     return (
       <div ref={this.divRef} className={containerClass} style={style} data-number={msg.number}>
-        {dayBreak && !isFirstUnread ? <DayBreak when={msg.when} /> : null}
+        {dayBreak && !isLastRead ? <DayBreak when={msg.when} /> : null}
         {renderSigil
           ? <MessageWithSigil {...messageProps} />
           : <MessageWithoutSigil {...messageProps} />}
-        {isFirstUnread
-          ? <UnreadMarker ref={unreadRef} dayBreak={dayBreak} when={msg.when} style={{ marginTop: (renderSigil ? "-17px" : "-6px") }} />
-          : null}
+        <div className="f9 relative w-100 overflow-hidden" style={unreadContainerStyle}>{isLastRead
```

Comment:
you can get similar results with `<Box fontSize='0' position='relative' width='100%' overflow='hidden' style={unreadContainerStyle}>` from indigo-react. Why? Well we just want to phase out the atomic CSS and it makes less for us to rewrite later. Just stuff to consider when writing these divs.